<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1031</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Resources/1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Next">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px ;
	background-color: rgb(0, 255, 252);
	color: rgb(0, 0, 0);
}
QPushButton:hover {
	background-color: rgb(66, 82, 91);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {
	background-color: rgb(66, 82, 91);
}
</string>
   </property>
   <property name="text">
    <string>► Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Previous">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px ;
	background-color: rgb(66, 82, 91);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(0, 255, 252);
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(0, 255, 252);
}
</string>
   </property>
   <property name="text">
    <string>Previous ◄</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dashboard_table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: rgb(10, 179, 179);
    color: white; /* Text color */
    border: 2px solid #005c5c; /* Border color */
    font-size: 12px; /* Font size */
}

QTableWidget::item {
    border: 1px solid #ffffff; /* Item border color */
}

QHeaderView::section {
    background-color: rgb(20, 130, 130); /* Header background color */
    color: white; /* Header text color */
    padding: 5px;
    font-size: 12px; /* Header font size */
}

QTableWidget::item:selected {
    background-color: rgb(0, 100, 100); /* Background color for selected items */
    color: white; /* Text color for selected items */
}

QSlider {
    background-color: rgb(10, 179, 179); /* Background color of the slider track */
    height: 10px; /* Height of the slider track */
}

QSlider::groove:horizontal {
    background: rgba(255, 255, 255, 0.5); /* Color of the groove */
    height: 5px; /* Height of the groove */
}

QSlider::handle:horizontal {
    background: rgb(20, 130, 130); /* Color of the slider handle */
    border: 2px solid rgb(255, 255, 255); /* Border of the slider handle */
    width: 20px; /* Width of the slider handle */
    margin: -10px 0; /* Center the handle on the groove */
    border-radius: 10px; /* Rounded corners */
}

QSlider::handle:horizontal:hover {
    background: rgb(30, 160, 160); /* Color of the slider handle on hover */
}

QSlider::sub-page:horizontal {
    background: rgb(0, 100, 100); /* Color of the filled part of the slider */
}

QSlider::add-page:horizontal {
    background: rgba(255, 255, 255, 0.3); /* Color of the unfilled part of the slider */
}

</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>122</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>32</number>
   </attribute>
   <column>
    <property name="text">
     <string>Deptor Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Dept Owed</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Overdue Payments</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="owed_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Total Owed:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="chart_layout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="Title_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DASHBOARD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>871</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
